content/influxdb/cloud-serverless/process-data/downsample.md

```diff
@@ -85,17 +85,17 @@ import pandas
 # Instantiate an InfluxDBClient3 client configured for your unmodified bucket
 influxdb_raw = InfluxDBClient3(
     host='cloud2.influxdata.com',
-    org='ORG_NAME',
     token='API_TOKEN',
     database='RAW_BUCKET_NAME'
 )
 
-# Instantiate an InfluxDBClient3 client configured for your downsampled bucket
+# Instantiate an InfluxDBClient3 client configured for your downsampled database.
+# When writing, the org= argument is required by the client (but ignored by InfluxDB).
```

Comment:
I think the v3 clients let you omit the org. Like I mentioned above, I'm pretty sure I tested this.